### PR TITLE
gmt5: update to 5.4.5

### PR DESCRIPTION
**Pull request by seisman**
###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.5 18F132
Xcode 10.3 10G8

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [X] checked your Portfile with `port lint`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by seisman** [comment]
I also submitted a new port gmt6 in PR #5225. gmt6 conflicts with gmt5. I'll add a "conflict" line if the gmt6 port is approved.

File: science/gmt5/Portfile [hunk 1]
```diff
@@ -30,12 +29,12 @@ use_xz              yes
 distname            gmt-${version}
 distfiles           ${distname}-src${extract.suffix}
 
-# stealth update 
+# stealth update
 dist_subdir         ${name}/${version}_1
```
Comment (mojca) [inline science/gmt5/Portfile]: You may now remove the `_1` part. Or the full line altogether.

Comment (seisman) [reply science/gmt5/Portfile]: I just removed the dist_subdir line and added a "conflicts gmt6" line.

**Comment by seisman** [comment]
@mojca This PR is ready for review.

**Comment by cjones051073** [comment]
Merging under maintainer timeout

**Pull request merged**